this is for the python workshop.

zhou-waner opened the pull request:
I edited the school.py

rehap98 reviewed (changes requested):
Nice! You could take this a step further by randomly selecting a college or class at BU. Also, could you please remove the git workshop file from this pull request? It is usually good practice to separate different assignments into different pull requests to keep the review process organized.

zhou-waner commented:
Hi,

I just removed the git-workshop related file as you suggested! 
please check #36 PR for my git-workshop practice!

Also, I have added randomness. I think I have finished both git and python workshop.
Please contact me if you have any questions!

zhou-waner commented:
> Nice! You could take this a step further by randomly selecting a college or class at BU. Also, could you please remove the git workshop file from this pull request? It is usually good practice to separate different assignments into different pull requests to keep the review process organized.
